spec/controllers/idv/in_person/state_id_controller_spec.rb

```diff
@@ -225,15 +225,16 @@
             expect(subject.user_session['idv/in_person'][:pii_from_user]).to_not have_key attr
           end
 
-          make_pii
+          build_pii_before_state_id_update
 
-          # pii includes address attrs on re-visiting state id pg
+          # since same_address_as_id was initially true, pii includes residential address attrs,
+          # which are the same as state id address attrs, on re-visiting state id pg
           expect(subject.user_session['idv/in_person'][:pii_from_user]).to include(
-            address1:,
-            address2:,
-            city:,
-            state:,
-            zipcode:,
+            identity_doc_address1:,
+            identity_doc_address2:,
+            identity_doc_city:,
+            identity_doc_address_state:,
+            identity_doc_zipcode:,
```

Comment:
This test looks like it checks the data before the PUT. What do you think about confirming that the residential address data is included in pii_from_user at this point?